apps/evm/src/pages/Governance/ProposalList/index.tsx

```diff
@@ -1,87 +1,177 @@
+    { enabled: !!latestProposalData?.proposalId },
+  );
+
   const createProposalEnabled = useIsFeatureEnabled({ name: 'createProposal' });
   const { navigate } = useNavigate();
   const { newProposalStep } = useParams<{
     newProposalStep: 'create' | 'file' | 'manual' | undefined;
   }>();
   const [showCreateProposalModal, setShowCreateProposalModal] = useState(!!newProposalStep);
-  const { t } = useTranslation();
-  const styles = useStyles();
+
+  const handleSearchInputChange: InputHTMLAttributes<HTMLInputElement>['onChange'] = changeEvent =>
+    setSearchValue(changeEvent.currentTarget.value);
+
+  // User has enough voting weight to create proposal and doesn't currently have an active or
+  // pending proposal
+  const canCreateProposal =
+    currentVotesData?.votesMantissa.isGreaterThanOrEqualTo(CREATE_PROPOSAL_THRESHOLD_MANTISSA) &&
+    latestProposalStateData?.state !== 0 &&
+    latestProposalStateData?.state !== 1;
 
   return (
-    <div css={styles.root}>
-      <div css={[styles.header, styles.bottomSpace]}>
-        <Typography variant="h4">{t('vote.proposals')}</Typography>
+    <div className={cn(className, 'space-y-4 md:space-y-6')}>
+      <div className="flex justify-between items-end">
+        <h4 className="text-lg">{t('vote.proposals')}</h4>
 
         {createProposalEnabled && (
-          <div css={styles.createProposal} data-testid={TEST_IDS.createProposal}>
+          <div className="flex items-center" data-testid={TEST_IDS.createProposal}>
             <TextButton
               onClick={() => {
                 setShowCreateProposalModal(true);
                 navigate(routes.governanceProposalCreate.path);
               }}
-              css={styles.marginLess}
+              className="p-0 h-7 mr-2"
               disabled={!canCreateProposal}
             >
               {t('vote.createProposalPlus')}
             </TextButton>
 
-            <InfoIcon tooltip={t('vote.requiredVotingPower')} css={styles.infoIconWrapper} />
+            <InfoIcon tooltip={t('vote.requiredVotingPower')} />
           </div>
         )}
       </div>
 
-      {isLoading && <Spinner css={styles.loader} />}
+      <div className="space-y-4 sm:flex sm:gap-x-6 sm:space-y-0 sm:justify-between">
+        <Select
+          label={t('vote.proposalStateFilter.label')}
+          variant="secondary"
+          placeLabelToLeft
+          options={selectOptions}
+          className="min-w-[230px]"
+          value={selectedProposalState}
+          onChange={newValue =>
+            setSelectedProposalState(newValue as ProposalState | typeof ALL_OPTION_VALUE)
+          }
+        />
+
+        <TextField
+          isSmall
+          value={searchValue}
+          onChange={handleSearchInputChange}
+          placeholder={t('vote.searchInput.placeholder')}
+          leftIconSrc="magnifier"
+          variant="secondary"
+          className="sm:max-w-[300px] w-full"
+        />
+      </div>
```

Comment:
Should this be behind a a feature flag for now? I'm wondering about UI releases while the search is sorted out by the subgraph

Reply:
You're right, good catch. Let me make a feature flag for this.